config-connector/solutions/sql/kpt/mysql-public/README.md

```diff
@@ -0,0 +1,49 @@
+MySQL Public
+==================================================
+# NAME
+  mysql-public
+# SYNOPSIS
+  Config Connector compatible yaml files to configure a public mySQL cluster
+# CONSUMPTION
+  Download the package using [kpt](https://googlecontainertools.github.io/kpt/).
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/sql/kpt/mysql-public mysql-public
+  ```
+# SETTERS
```

Comment:
According to the [CFT example](https://github.com/terraform-google-modules/terraform-google-sql-db/tree/master/examples/mysql-public), database name is not a settable field, while the authorized networks are.
I'd suggest that
1. Let's remove the database-name setter.
2. Add one setter for authorized network name, and another for the value.